src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.DiagnosticState.cs

```diff
@@ -16,303 +16,305 @@
 
 namespace Microsoft.CodeAnalysis.Diagnostics.EngineV1
```

Comment:
most of change here is formatting change. somehow (merge?), formatting of this file was all messed up.